.github/workflows/c-tests.yml

```diff
@@ -49,16 +49,13 @@ jobs:
       - name: Install RTI
         uses: ./.github/actions/install-rti
         if: ${{ runner.os == 'macOS' || runner.os == 'Linux' }}
-      - name: Install Arduino Platform for MacOS
+      - name: Install Arduino CLI for MacOS
         run: |
-          brew install --cask arduino
+          brew install arduino-cli
         if: ${{ runner.os == 'macOS' }}
-      - name: Install Arduino Platform for Linux
+      - name: Install Arduino CLI for Linux
         run: |
-          wget "https://downloads.arduino.cc/arduino-1.8.19-linux32.tar.xz"
-          tar -xvf arduino-1.8.19-linux32.tar.xz
-          cd arduino-1.8.19/
-          sudo ./install.sh
+          curl -fsSL https://raw.githubusercontent.com/arduino/arduino-cli/master/install.sh | sh
```

Comment:
It looks like this installs the latest `arduino-cli` from `master`, which could cause unexpected problems when something in their `master` branch changes that breaks compatibility with us. It might be better to pin `arduino-cli` to a specific version number.

Reply:
This was on their how to install arduino-cli guide for non-Mac systems. I'll try to point it to the current version that works.